java/google/registry/whois/WhoisModule.java

```diff
@@ -42,4 +47,11 @@ static Reader provideHttpInputReader(HttpServletRequest req) {
       throw new RuntimeException(e);
     }
   }
+
+  @Provides
+  @Config("whoisCommandFactory")
+  static WhoisCommandFactory provideWhoisCommandFactory(
+    @Config("whoisCommandFactoryClass") String factoryClass) {
```

Comment:
The line-wrapping/indentation is wrong.  Use google-java-format on this block to fix.